src/modules/dashboard/components/ColonyFundingMenu/ColonyFundingMenu.tsx

```diff
@@ -80,30 +76,45 @@ const ColonyFundingMenu = ({
     [colony, openTokensMoveDialog, selectedDomainId],
   );
 
+  const canEdit =
+    userHasRole(rootRoles, ColonyRole.Root) ||
+    userHasRole(rootRoles, ColonyRole.Administration);
+  const canMoveTokens = userHasRole(rootRoles, ColonyRole.Funding);
+
+  const isSupportedColonyVersion =
+    parseInt(version, 10) >= ColonyVersion.CeruleanLightweightSpaceship;
+  const isNetworkAllowed = !!ALLOWED_NETWORKS[networkId || 1];
```

Comment:
Yes, which is ethereum mainnet: 

https://github.com/JoinColony/colonyDapp/blob/f53547841df4800f4a2d93d1b001027779a4efd7/src/modules/constants.ts#L98-L103

Since `ETHEREUM_NETWORK.chainId` is `1`